src/coreclr/src/jit/dll/CMakeLists.txt

```diff
@@ -2,17 +2,25 @@ project(ClrJit)
 
 set_source_files_properties(${JIT_EXPORTS_FILE} PROPERTIES GENERATED TRUE)
 
+add_library_clr(clrjit_static
+    STATIC
+    ${SHARED_LIB_SOURCES}
+    ${JIT_ARCH_SOURCES}
+    ${JIT_DEF_FILE}
```

Comment:
Right. I think this is not necessary. Previously we were adding def files to the static lib, since they were together with other sources. Now we do not have to.